.circleci/config.yml

```diff
@@ -32,6 +32,55 @@ jobs:
           path: build/test-results
       - store_artifacts:
           path: build/reports/tests
+  end-to-end-tests:
+    executor:
+      name: hmpps/java
```

Comment:
Any particular reason to use a java image/executor here?  I'm assuming it's a pretty hefty machine image.

You could probably get away with `hmpps/default_small` as you're just running bash scripts.